internal/db/mock_gen.go

```diff
@@ -0,0 +1,214 @@
+// Copyright 2020 The Gogs Authors. All rights reserved.
```

Comment:
This is renamed from `mocks.go`, which is now used for storing generated code for mocks.